lib/ui/list_item.dart

```diff
@@ -9,34 +11,76 @@ class ResultListItem extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    final textStyle = TextStyle(fontWeight: FontWeight.normal, fontSize: TextSize.mediumTitle);
+    final pointsStyle = TextStyle(
+      fontWeight: FontWeight.w700,
+      fontFamily: FontFamily.roboto,
+      fontSize: TextSize.mediumTitle,
+      color: Colors.white,
+    );
+
+    final pktStyle = TextStyle(
+      fontWeight: FontWeight.w700,
+      fontFamily: FontFamily.roboto,
+      fontSize: 9,
+      color: Colors.white,
+    );
+
     return _ListItem(
-      child: Column(
-      children: [
-        Expanded(
-          child: Align(
-            alignment: Alignment.centerLeft,
-            child: Padding(
-              padding: EdgeInsets.all(4.0),
-              child: Text(searchResult.name!, style: textStyle,)
-            )
-          )
-        ),
-        Align(
-          alignment: Alignment.bottomCenter,
-          child: ClipRRect(
-            borderRadius: BorderRadius.only(
-              bottomLeft: Radius.circular(5),
-              bottomRight: Radius.circular(5)
+      child: Row(
+        crossAxisAlignment: CrossAxisAlignment.center,
+        children: [
+          Container(
+            width: 40,
```

Comment:
Te 40 to najlepiej wynieść do stałej. `final leftBoxSize = 40.0` Wtedy zamiast 40 tu i niżej użyć to. A w borderRasius zamiast `Radius.circular(20)` to `Radius.circular(leftBoxSize / 2)`. Jak się zmieni wielkość to będzie jedna zmiana.